TESTS/mbedmicro-rtos-mbed/timer/main.cpp

```diff
@@ -24,7 +24,7 @@ void blink(void const *n) {
 }
 
 int main(void) {
-    GREENTEA_SETUP(total_ticks + 5, "timing_drift_auto");
+    GREENTEA_SETUP(total_ticks + 5, "default_auto");
```

Comment:
This is an incorrect change, the host test should be set to `timing_drift_auto`. Please revert this.